### DISCUSSION TITLE
Change window title bar color

### DISCUSSION THREAD

**Discussion by eamonburns** [r0x0r/pywebview#1279] [Q&A] [answered]
Is there a way to change the color of the title bar? I want it to be a darker gray color to go well with dark mode.

**Comment by Git002** [comment]
Yeah that would be so good

**Reply by eamonburns** [reply]
So it isn't possible?

**Reply by Git002** [reply, marked as answer]
Nah man, you can set the backgroud color of the window but not the title bars as black. Though a workaround would be to make your own cross and minimize button in the html and link it to python (which is doable) will do your job...

**Reply by r0x0r** [reply]
Changing title bar color or dark mode is not implemented. Contributions are much welcomed.
